Execution Result for model_h2o_prediction.py
--------------------------------------------------
Execution error (not missing module). Attempts: 1
LOG_START:Changed working directory to: /workspace/pipeline_c4817250-b02b-49ac-a00b-e7f87e432343:LOG_END
Checking whether there is an H2O instance running at http://localhost:54321..... not found.
Attempting to start a local H2O server...
  Java Version: openjdk version "21.0.8" 2025-07-15; OpenJDK Runtime Environment (build 21.0.8+9-Debian-1); OpenJDK 64-Bit Server VM (build 21.0.8+9-Debian-1, mixed mode, sharing)
  Starting server from /usr/local/lib/python3.11/site-packages/h2o/backend/bin/h2o.jar
  Ice root: /tmp/tmpcakku5mm
  JVM stdout: /tmp/tmpcakku5mm/h2o_unknownUser_started_from_python.out
  JVM stderr: /tmp/tmpcakku5mm/h2o_unknownUser_started_from_python.err
  Server is running at http://127.0.0.1:54321
Connecting to H2O server at http://127.0.0.1:54321 ... successful.
Warning: Your H2O cluster version is (5 months and 7 days) old.  There may be a newer version available.
Please download and install the latest version from: https://h2o-release.s3.amazonaws.com/h2o/latest_stable.html
--------------------------  ----------------------------------
H2O_cluster_uptime:         01 secs
H2O_cluster_timezone:       Etc/UTC
H2O_data_parsing_timezone:  UTC
H2O_cluster_version:        3.46.0.7
H2O_cluster_version_age:    5 months and 7 days
H2O_cluster_name:           H2O_from_python_unknownUser_unrgd9
H2O_cluster_total_nodes:    1
H2O_cluster_free_memory:    1007 Mb
H2O_cluster_total_cores:    16
H2O_cluster_allowed_cores:  2
H2O_cluster_status:         locked, healthy
H2O_connection_url:         http://127.0.0.1:54321
H2O_connection_proxy:       {"http": null, "https": null}
H2O_internal_security:      False
Python_version:             3.11.13 final
--------------------------  ----------------------------------
LOG_START:H2O initialized successfully:LOG_END
LOG_START:Model loaded successfully: DRF_1_AutoML_1_20250904_154856:LOG_END
LOG_START:Generating 30 future periods with frequency D:LOG_END
LOG_START:Model expects features: ['fecha__year', 'fecha__month', 'fecha__day', 'fecha__dayofweek', 'fecha__is_weekend', 'fecha__is_month_start', 'fecha__is_month_end', 'monto_total_lag_1', 'monto_total_lag_2', 'monto_total_lag_3', 'monto_total_lag_7']:LOG_END
Parse progress: |████████████████████████████████████████████████████████████████| (done) 100%
drf prediction progress: | (failed)
ERROR_START:Job with key $03017f00000132d4ffffffff$_bb721f95ac073dc4279faeddfe6547f2 failed with an exception: java.lang.IllegalArgumentException: Test/Validation dataset has categorical column 'monto_total_lag_1' which is real-valued in the training data stacktrace:  java.lang.IllegalArgumentException: Test/Validation dataset has categorical column 'monto_total_lag_1' which is real-valued in the training data 	at hex.Model.adaptTestForTrain(Model.java:1833) 	at hex.Model.adaptTestForTrain(Model.java:1643) 	at hex.Model.adaptTestForTrain(Model.java:1639) 	at hex.Model.adaptFrameForScore(Model.java:1981) 	at hex.Model.score(Model.java:1999) 	at water.api.ModelMetricsHandler$1.compute2(ModelMetricsHandler.java:555) 	at water.H2O$H2OCountedCompleter.compute(H2O.java:1704) 	at jsr166y.CountedCompleter.exec(CountedCompleter.java:468) 	at jsr166y.ForkJoinTask.doExec(ForkJoinTask.java:263) 	at jsr166y.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:976) 	at jsr166y.ForkJoinPool.runWorker(ForkJoinPool.java:1479) 	at jsr166y.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:104) :ERROR_END
H2O session _sid_8571 closed.
LOG_START:H2O cluster shutdown:LOG_END

